src/components/form-helpers/form-helpers.js

```diff
@@ -136,17 +138,27 @@ export const FieldGroup = ({ className, children }) => (
   <ul className={concatClasses('field-group', className)}>{children}</ul>
 );
 
+const SetFormMessage = connect(({ formik: { setErrors }, error }) => {
+  useEffect(() => {
+    if (error) {
+      setErrors(error);
+    }
+  }, [error, setErrors]);
+
+  return null;
+});
+
 export const FormErrors = connect(
   /**
    * @param { Object } props
    * @param { import('formik').FormikProps<Values> } props.formik
    */
   ({ formik: { errors } }) => {
     const formError =
-      errors && errors['_general']
-        ? { message: errors['_general'], className: 'dp-error' }
-        : errors && errors['_generalWarning']
-        ? { message: errors['_generalWarning'], className: 'dp-warning-message' }
+      errors && errors['_error']
```

Comment:
what do you think about renaming some other names that refer to _errors_ in place of _messages_?

![image](https://user-images.githubusercontent.com/1157864/61234913-a5855100-a70a-11e9-853d-3c6a0fe70596.png)

Reply:
Yes! i changed